README.md

```diff
@@ -18,6 +18,12 @@ TODO:
 ## Deploy Shared Infrastructure
 Before you can deploy any of the other stacks, you must deploy some prerequisite pieces of shared infrastructure. These are required by both the application components and the CI/CD stacks that test and deploy those application components.
 
+In all cases below, deployment is done using AWS CLI commands. The examples given must be rewritten with valid values for each of the parameters. The commands assume that default parameters are chosen in each case. In most deployments, however, non-default parameters will be desired. In each case, parameters defaults can be overwritten by adding a line similar to:
```

Comment:
Actually, that statement is not overly valid since I added some (more) parameter override examples in the commands.

Reply:
Which statement isn't overly valid? My comment, or what's in the README?

Reply:
What is in the README. Since I later added parameter overrides to most of the example/default command lines, the statement "The commands assume that default parameters are chosen in each case." no longer seems fully valid. Initially, I was trying to keep things generic, as it was earlier. But, then, it became clear that some values needed to be added/overridden. Most specifically the SubDomain of the static host since the same template is used for multiple things, that value cannot be left empty. The template has no default and if it did, the second run would not complete.